invenio_sip2/helpers.py

```diff
@@ -25,19 +25,21 @@
 class MessageTypeFixedField(object):
     """SIP2 Message type fixed field helper class."""
 
-    def __init__(self, field_id, length, label):
+    def __init__(self, name, field):
         """Constructor."""
-        self.field_id = field_id
-        self.length = length
-        self.label = label
+        self.field_id = name
+        self.length = field.get('length')
+        self.fill = field.get('fill', ' ')
+        self.label = field.get('label')
 
     def __str__(self):
         """String representation of fixed field."""
         return 'MessageTypeVariableField() field_id={field_id} ' \
```

Comment:
You are sure you like to return: `MessageTypeVariableField() ...`

Reply:
Yes, it's helpful for devel.